packages/gateway/tests/integration/connection.spec.ts

```diff
@@ -120,7 +119,7 @@ const createUws = async (options: {
         }
         // retrieve listening port
         uwsToken = token
-        port = uWS.us_socket_local_port(token)
+        port = uWS.us_socket_local_port(uwsToken)
         resolve({ port, uwsToken })
```

Comment:
```suggestion
        resolve({ port, uwsToken: token })
```

Reply:
For some reason GitHub doesn't allow me to select three lines and make one suggestion, so please check the other two above